src/engraving/libmscore/measure.cpp

```diff
@@ -4408,6 +4411,16 @@ double Measure::computeFirstSegmentXPosition(Segment* segment)
         // The start-repeat should overlap the end-repeat of the previous measure
         x -= score()->styleMM(Sid::endBarWidth);
     }
+    // Do a final check of chord distances (invisible items may in some cases elude the 2 previous steps)
+    if (segment->isChordRestType()) {
+        double barNoteDist = static_cast<double>(score()->styleMM(Sid::barNoteDistance));
```

Comment:
I think the preferred way is calling `.val()`: 
`double barNoteDist = score()->styleMM(Sid::barNoteDistance).val()`.

Reply:
@mike-spa It seems like this comment got overlooked :)